Frontend/nutri-pal/src/app/gates/login/page.tsx

```diff
@@ -8,6 +8,7 @@ import { useEffect, useState } from 'react';
 import { Inputs } from '@/components/gates/Inputs';
 import { auth } from '@/app/firebase-config';
 import { signInWithEmailAndPassword } from 'firebase/auth';
+import Image from 'react';
```

Comment:
nit: let's try to keep changes like this for a separate PR b/c its unrelateing to init database